README.md

```diff
@@ -1,2 +1,17 @@
 # knex-mysql-deadlock
 A custom knex mysql dialect, which does deadlock retries
+
+## Install
+
+```bash
+npm install knex-mysql-deadlock
```

Comment:
to go with the peerDeps changes, the install prompt should include the required peerdeps, eg:

```
npm install knex node-mysql knex-mysql-deadlock
```